homeassistant/components/nederlandse_spoorwegen/strings.json

```
@@ -0,0 +1,144 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Nederlandse Spoorwegen Setup",
+        "description": "Set up your Nederlandse Spoorwegen integration.",
+        "data": {
+          "api_key": "API key"
+        },
+        "data_description": {
+          "api_key": "Your NS API key."
+        }
+      },
+      "routes": {
+        "title": "Add Route",
```

Sentence-casing for all strings:

```suggestion
        "title": "Add route",
```